pkg/upgradepatch/hc.cr.json

```diff
@@ -53,5 +53,7 @@
     "deployVmConsoleProxy": false,
     "enableApplicationAwareQuota": false
   },
-  "status": {}
+  "status": {
+    "nodeInfo": {}
+  }
```

Comment:
why we need to specifically mention this stanza in the status? auto-generated?

Reply:
auto generated, yes.